docs/get-started/roles-and-permissions/lbac.md

```diff
@@ -0,0 +1,35 @@
+# Labels for access control in PMM
+
+
+Label-based access control in PMM allows you to manage who has access to metrics based on labels. By creating roles, you can specify which data can be queried based on specific label criteria, for instance, allowing the QA team to view data related to test environments. 
+ 
+With Label-based access control, you can associate multiple labels with a role, ensuring only data from series that match your defined labels is returned. 
+
+
+## Predefined vs Custom labels
+
+PMM supports predefined as well as custom labels. PMM always assigns predefined labels. When an object (Node, Service, or Agent) is created, a user can set initial values.  Predefined labels are automatically assigned and updated by PMM. Custom labels are assigned and updated only by a user.
+
+**Examples**
+
+
+| **Label Type**| **Object**| **Label name **| **Example** |                                                                                                
+|----------|--------|-------|------------------------------|
+| **Standard**  | Node   | node_id |/node_id/123|                                          
```

Comment:
We use inconsistent language - standard vs predefined labels

Reply:
let's use the wording `standard` since we have been using it.

Reply:
Fixed.